### PR TITLE
added english language

### PR DESCRIPTION
**Pull request by gekigek99**

### CODE REVIEW

**Comment by gekigek99** [comment]
I changed just one file because I coud not find the function where the language is set. So it still needs to be specified that "en.json" exists.

**Comment by BolverBlitz** [comment]
@gekigek99 I'll demand Chances then 😝

**Comment by geozukunft** [comment]
@BolverBlitz He probably meant changes

File: Finanzen/Telegram/lang/en.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    "Antworten": {
+        "Ja": "Yes",
+        "Nein": "No",
+        "Vollständig": "Entirely",
+        "Teil": "Partial vaccinated",
+        "NichtSagen": "I don’t want to say"
+    },
+    "Knöpfe": {
+        "Reg": "Registration",
+        "WebReg": "This",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:24]: translation?

File: Finanzen/Telegram/lang/en.json [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            "Artikel": "Buy items",
+            "Spenden": "Donations",
+            "Other": "More Info",
+            "Webpanel": "Webpanel"
+        }
+    },
+    "Payed": {
+        "Text": "{{Username}}, {{Status_String}}\n\nAmount: {{money}}\nID: {{PayedID}}",
+        "True": "your payment was made by our system <b>confirmed</b>",
+        "False": "your payment has not been made by our system yet <b>not confirmed</b>. Should this still be the case at the beginning of the LAN party please contact the organizers!"
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json]: translation?

File: Finanzen/Telegram/lang/en.json [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        "Text": "<b>More information:</b>\n{{Username}} here you will find more useful information.",
+        "Knöpfe": {
+            "Rules": "Rules",
+            "Legal": "Legal",
+            "Sprache": "Language",
+            "Hauptmenu": "⏪ Main menu"
+        },
+        "Regeln": {
+            "Exit": "❌ Exit",
+            "AcceptedDate": "\nYou have the rules on {{akteptiert}} file"
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json]: translation/contex?

File: Finanzen/Telegram/lang/en.json [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+        },
+        "Legal": {
+            "Exit": "❌ Exit",
+            "AcceptedDate": "\nYou have filed the notices and provisions on {{akteptiert}} file"
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json]: translation/contex?

File: Finanzen/Telegram/lang/en.json [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+        }
+    },
+    "Regeln": {
+        "Text": "<b>Please read the house rules carefully!</b>\n\n <i>The following rules must be strictly adhered to for the duration of the LAN, a violation of which may result in the discretion until exclusion from the LAN.</i>",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:126]: translation?

File: Finanzen/Telegram/lang/en.json [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+        "RegelnAnzahl": "7",
+        "Regeln": {
+            "0": "1: There is no exception to the strict smoking ban in the building, there are aerosols -> it is <b>Forbidden!</b> For smokers there is the possibility to smoke like last year in the courtyard with <u>closed building door</u>.",
+            "1": "2: According to current regulations, an FFP2 mask requirement applies on the ground floor during office hours (7-17 pm Weekdays)",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:130]: translation?

File: Finanzen/Telegram/lang/en.json [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+            "2": "3: <b>EVERYONE</b> clear <b>AUTONOMOUSLY</b> their dishes and garbage. (For a fee of 150€ you get your own VIP dishes and GEOs Dad clears your garbage and dishes.)",
+            "3": "4: Remaining unused food in the residual waste. Pizzas have no place in their boxes in the cardboard boxes. Violation of this rule will result in a penalty fee of 150€. (PS: Yes this rule exists for good reason.)",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:132]: translation?

File: Finanzen/Telegram/lang/en.json [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+            "Zustimmen": "Accept",
+            "Ablehnen": "Reject"
+        },
+        "Antworten": {
+            "Zustimmen": "Accept",
+            "Ablehnen": "Reject"
+        }
+    },
+    "loadprice": {
+        "Text": "{{Amount}} item were read or updated and this took {{duration}}"
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:165]: translation?

File: Finanzen/Telegram/lang/en.json [hunk 9]
```diff
@@ -0,0 +1,206 @@
...
+            "Bestätigen": "Confirm",
+            "Kaufen": "Booked"
+        },
+        "Confirm": {
+            "NotPayed": "You have not yet paid or your payment has not yet been processed.",
+            "DoesNotExist": "The selected product {{produktname}} by {{produktcompany}} was unfortunately removed from the offer.",
+            "NotEnoth": "The product {{produktname}} by {{produktcompany}} is no longer available in sufficient quantities!",
+            "OtherBuy": "The selected product was bought at the same time by someone else, please try again"
+        },
+        "MainMessage": "<b>Selected item:</b> {{produktname}} by {{produktcompany}}\nPrice per piece: {{price}}\nIn stock: {{storrage}}\n\nPlease select the quantity below and confirm your purchase. Please wait for confirmation from the bot.",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:197]: spelling error "storrage"

File: Finanzen/Telegram/lang/en.json [hunk 10]
```diff
@@ -0,0 +1,206 @@
...
+        "NotFound": {
+            "ID": "NotFound",
+            "Text": "Item could not be found",
+            "Message": "The item {{}} could not be found."
+        },
+        "Found": {
+            "ID": "N{{ID}}",
+            "Text": "{{Produkt}} by {{Hersteller}}",
+            "Beschreibung": "Price per piece: {{Preis}} - There are {{Available}} in stock",
+            "Message": "<b>Selected item:</b> {{produktname}} by {{produktcompany}}\nPrice per piece: {{price}}\nIn stock: {{storrage}}\n\nTo buy the item click on the button below."
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:182]: spelling error "storrage"

File: Finanzen/Telegram/lang/en.json [hunk 11]
```diff
@@ -0,0 +1,206 @@
...
+            "Kaufen": "Booked"
+        },
+        "Confirm": {
+            "NotPayed": "You have not yet paid or your payment has not yet been processed.",
+            "DoesNotExist": "The selected product {{produktname}} by {{produktcompany}} was unfortunately removed from the offer.",
+            "NotEnoth": "The product {{produktname}} by {{produktcompany}} is no longer available in sufficient quantities!",
+            "OtherBuy": "The selected product was bought at the same time by someone else, please try again"
+        },
+        "MainMessage": "<b>Selected item:</b> {{produktname}} by {{produktcompany}}\nPrice per piece: {{price}}\nIn stock: {{storrage}}\n\nPlease select the quantity below and confirm your purchase. Please wait for confirmation from the bot.",
+        "MainProgress": "Booking menu is called up...",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:198]: translation?

File: Finanzen/Telegram/lang/en.json [hunk 12]
```diff
@@ -0,0 +1,206 @@
...
+        "Escape": "Booking menu has been closed...",
+        "TooMutch": "You can’t buy more than we have in stock.",
+        "TooLitle": "You can’t buy less than one",
+        "Donation": "Thank you for your donation of {{price}} to EBG. PW ❤️",
+        "ItemBought": "You have successfully bought {{amount}} times the product {{produktname}} by {{produktcompany}} for {{price}}.\n\nYou can now view the item.",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:203]: translation?

File: Finanzen/Telegram/lang/en.json [hunk 13]
```diff
@@ -0,0 +1,206 @@
...
+        "Confirm": {
+            "NotPayed": "You have not yet paid or your payment has not yet been processed.",
+            "DoesNotExist": "The selected product {{produktname}} by {{produktcompany}} was unfortunately removed from the offer.",
+            "NotEnoth": "The product {{produktname}} by {{produktcompany}} is no longer available in sufficient quantities!",
+            "OtherBuy": "The selected product was bought at the same time by someone else, please try again"
+        },
+        "MainMessage": "<b>Selected item:</b> {{produktname}} by {{produktcompany}}\nPrice per piece: {{price}}\nIn stock: {{storrage}}\n\nPlease select the quantity below and confirm your purchase. Please wait for confirmation from the bot.",
+        "MainProgress": "Booking menu is called up...",
+        "Escape": "Booking menu has been closed...",
+        "TooMutch": "You can’t buy more than we have in stock.",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:200]: added "we", check if it is correct

File: Finanzen/Telegram/lang/en.json [hunk 14]
```diff
@@ -0,0 +1,206 @@
...
+        "TooLitle": "You can’t buy less than one",
+        "Donation": "Thank you for your donation of {{price}} to EBG. PW ❤️",
```
Comment (gekigek99) [inline Finanzen/Telegram/lang/en.json:202]: what does "PW" means?

**Review by BolverBlitz — APPROVED** [review]

**Comment by BolverBlitz** [comment]
Please add english to line 104 on file Finanzen/Telegram/lang/de.json

**Comment by BolverBlitz** [comment]
Please add the following line to Finanzen/Telegram/src/telegram.js
Below line: https://github.com/EBG-PW/LAN-Abrechnung/blob/69a0057c92aac1ab9224df9fea0d7e397d08e036/Finanzen/Telegram/src/telegram.js#L189

Add: `bot.inlineButton(newi18n.translate('de', 'Sprachen.Knöpfe.EN'), {callback: 'lang_en'}),`

**Comment by gekigek99** [comment]
@BolverBlitz I added the requested corrections

**Pull request merged**